decentraland/components/presentational/__snapshots__/LinkToBlockchain.test.js.snap

```diff
@@ -2,13 +2,4 @@
 
 exports[`LinkToBlockchain renders the component null action 1`] = `""`;
 
-exports[`LinkToBlockchain renders the component valid action 1`] = `
-<Styled(Button)
-  onPress={[Function]}
-  transparent={true}
->
-  <Styled(Icon)
-    name="eye"
-  />
-</Styled(Button)>
-`;
+exports[`LinkToBlockchain renders the component valid action 1`] = `""`;
```

Comment:
Hmm, why is a valid action having this render as `null`?

Reply:
The network was hardcoded using `ropsten` and I've changed that, for now, the link will only be rendered with the network is `ropsten`.

Reply:
That makes sense. I think we need to find a way for this test to show what a valid snapshot would look like in production, though.

1. So maybe we mock the network name for this test and override the config?
2. Or make sure that in the test env there's a different config than the development config?

Reply:
Yeah, since our tests aren't interacting with the blockchain I've changed the `jest` setup to use testnet config.

Reply:
Okay cool, thanks. Resolving this!